federatedscope/cl/dataloader/Cifar10.py

```diff
@@ -0,0 +1,154 @@
+import math
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.backends.cudnn as cudnn
+import torchvision
+import torchvision.transforms as T
+import torchvision.transforms.functional as TF
+from torch.utils.data import DataLoader, Dataset
+from torchvision.datasets import CIFAR10, CIFAR100
+import pickle as pkl
+import numpy as np
+from federatedscope.register import register_data
+from federatedscope.core.auxiliaries.splitter_builder import get_splitter
+
+
+
+class SimCLRTransform():
+    def __init__(self, is_sup, image_size=32):
```

Comment:
please provide docstring to describe your functions and classes.

Reply:
finish